compiler/src/iree/compiler/Codegen/LLVMGPU/LLVMGPUTileAndDistribute.cpp

```diff
@@ -65,8 +65,38 @@ static void populateTilingReductionPatterns(RewritePatternSet &patterns) {
           StringAttr::get(context, getWorkgroupMemoryMarker())},
       StringAttr::get(context, getWorkgroupKTiledMarker()));
   filter.setMatchByDefault();
-  TilingPatterns<linalg::MatmulOp, linalg::BatchMatmulOp,
-                 linalg::GenericOp>::insert(patterns, tilingOptions, filter);
+  TilingPatterns<linalg::MatmulOp, linalg::BatchMatmulOp, linalg::GenericOp,
+                 linalg::Conv2DNhwcHwcfOp,
+                 linalg::Conv2DNchwFchwOp>::insert(patterns, tilingOptions,
```

Comment:
(Was this function referenced anywhere before this change?)

Is the addition of conv here related to this change and/or does it need a test? I'm not too familiar with the flow on the LLVMGPU side so just checking.